tracer/src/Datadog.Trace/ExtensionMethods/SpanExtensions.cs

```diff
@@ -20,24 +20,6 @@ namespace Datadog.Trace.ExtensionMethods
     /// </summary>
     public static class SpanExtensions
     {
-        /// <summary>
-        /// Adds standard tags to a span with values taken from the specified <see cref="DbCommand"/>.
-        /// </summary>
-        /// <param name="span">The span to add the tags to.</param>
-        /// <param name="command">The db command to get tags values from.</param>
-        public static void AddTagsFromDbCommand(this ISpan span, IDbCommand command)
```

Comment:
I thought about making this internal the last time I was here, but I thought it could be a nice helper for users of manual instrumentation.